### DISCUSSION TITLE
Button Matrix Examples???

### DISCUSSION THREAD

**Discussion by RandyRyanG** [tttapa/Control-Surface#934] [Q&A]
Can someone point me in the right direction or post their own arduino code as an example for using the Button Matrix class?

**Comment by Trickster-git** [comment]
https://tttapa.github.io/Control-Surface-doc/Doxygen/dc/d05/Keyboard-Matrix-BCD_8ino-example.html
https://tttapa.github.io/Control-Surface-doc/Doxygen/d4/d53/NoteButtonMatrix_8ino-example.html

**Reply by RandyRyanG** [reply]
Thanks, I have already looked at those examples,  and they are using the "NoteButtonMatrix" Class. I am looking for and example for the "ButtonMatrix" class. My sketch already has the NoteButtonMatrix working and sending midi notes., I am trying to keep an array of timestamps so that I can calculate key velocity.
